components/more_channels/more_channels.jsx

```diff
@@ -121,6 +121,10 @@ export default class MoreChannels extends React.Component {
         this.searchTimeoutId = searchTimeoutId;
     }
 
+    setSearchResults = (channels) => {
```

Comment:
Out of scope for this PR, but it's too bad we can't filter this out server-side to improve performance.